<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Fatimah" failures="1" tests="5" name="Personal loan" time="0.101" errors="0" timestamp="2023-07-04T14:27:00 SGT">
  <testcase classname="testngtutorialbyrsa.day1" name="Demo" time="0.028">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testngtutorialbyrsa.day1.Demo(day1.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- Demo -->
  <testcase classname="testngtutorialbyrsa.day2" name="ploan" time="0.002"/>
  <testcase classname="testngtutorialbyrsa.day4" name="LoginAPIHomeLoan" time="0.001"/>
  <testcase classname="testngtutorialbyrsa.day4" name="MobileLoginHomeLoan" time="0.002"/>
  <testcase classname="testngtutorialbyrsa.day4" name="WebloginHomeLoan" time="0.003"/>
  <testcase classname="testngtutorialbyrsa.day1" name="SecondTest" time="0.0">
    <ignored/>
  </testcase> <!-- SecondTest -->
</testsuite> <!-- Personal loan -->
